static/app/components/metrics/mriSelect/index.tsx

```diff
@@ -248,7 +244,7 @@ export const MRISelect = memo(function MRISelect({
         const parsedMRI = parseMRI(metric.mri);
         const isDuplicateWithDifferentUnit = metricsWithDuplicateNames.has(metric.mri);
         const isUnresolvedExtractedMetric = isExtractedCustomMetric(metric);
-        const showProjectBadge = hasExtractionRules && selectedProjects.length > 1;
+        const showProjectBadge = selectedProjects.length > 1;
```

Comment:
Are you sure we still want to display the project badge? It appears it was only displayed when the feature flag was enabled

Reply:
The UX is much better with it. I tried the page without it, but it just feels wrong.